modules/liveIntentIdSystem.js

```diff
@@ -178,22 +178,10 @@ export const liveIntentIdSubmodule = {
       if (lipbid) {
         value.lipbid = lipbid
         delete value.unifiedId
-        result.lipb = value
-      }
-
-      // Lift usage of uid2 by exposing uid2 if we were asked to resolve it.
-      // As adapters are applied in lexicographical order, we will always
-      // be overwritten by the 'proper' uid2 module if it is present.
-      if (value.uid2) {
-        result.uid2 = { 'id': value.uid2 }
       }
 
-      if (value.bidswitch) {
-        result.bidswitch = { 'id': value.bidswitch }
-      }
-
-      if (value.medianet) {
-        result.medianet = { 'id': value.medianet }
+      if (Object.keys(value).length > 0) {
+        result.lipb = value
```

Comment:
This is breaking. Now, in case there is a `lipb` field in the result, there will not be necessarily a `lipbid`. If the general approach in this PR is approved, I will adjust the bid adapters accessing `lipbid`.

Reply:
@jdwieland8282 as fyi

Reply:
I don't understand what breaks or why, can you describe the issue a little more?

Reply:
@jdwieland8282 The "breaking" part is that in the current implementation the existence of `lipb` implies the existence `lipb.lipbid`. This would not be the case with the changes above. I though I saw bid adapters relying on that fact. Scanned through adapters using `lipb.lipbid` one more time and it looks like they are actually not affected by this change. Probably misread the code. Sorry for confusion.